apps/schools/domains/common/components/searchInput/styles/styles.module.scss

```diff
@@ -0,0 +1,19 @@
+.search {
+  font-size: 150%;
```

Comment:
why is the font size in %?

Reply:
These are migrated styles from another module.scss. While there are no adaptations, we are working with percentages